setup.py

```diff
@@ -1,2 +1,2 @@
 import os
-os.system("python3 ./utilities/svg_parsing/svg_parser.py > ./hs/SVGGen.hs")
+os.system("python ./utilities/svg_parsing/svg_parser.py > ./hs/SVGGen.hs")
```

Comment:
Yeah sorry I didn't know how to create a pull request JUST for the js file (is that even possible?). I'll remove the change right now.